src/components/header/Examples.tsx

```diff
@@ -47,26 +48,27 @@ export const Example: React.FC = () => {
 
   return (
     <Menu>
-      <Tooltip
-        shouldWrapChildren
-        hasArrow={!walletPublicKey}
-        label={
-          walletPublicKey
-            ? "Load an example transaction"
-            : "Please connect a wallet to contiune"
-        }
-      >
-        <MenuButton
-          as={Button}
-          rightIcon={<ChevronDownIcon />}
-          variant="ghost"
-          textColor="white"
-          isDisabled={!walletPublicKey}
+      <DarkMode>
```

Comment:
Are `DarkMode` required? In the preview, it's actually showing up as light mode so it must be ignoring it anyway?

![image](https://user-images.githubusercontent.com/4444588/184819272-70412fa5-e9a1-42a8-ad60-5a5dd9d19922.png)

If it wasn't ignoring these tags, it would have been dark background

Reply:
yes it is, otherwise the background for the button is on light mode when hover, we only want the menu list to be light mode


<img width="455" alt="Screen Shot 2022-08-16 at 5 34 12 pm" src="https://user-images.githubusercontent.com/111264560/184824033-52aecaa9-9a5d-40ed-9c12-bdd6310ab4b8.png">